sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py

```diff
@@ -24,6 +24,8 @@
 from ._helpers import ConversationHistory, Turn
 from ._utils import JsonLineChatProtocol
 
+if len(USER_AGENT) < 24:
```

Comment:
This check is not necessary at client side. RAISvc is the one who truncate the UA string if it's more than 24 chars, before pasing to AOAI. We can just pass the complete UA string to the server side.

Reply:
updated!